ecmd-core/ext/fapi2/pyapi/fapi2Attribute.i

```diff
@@ -90,9 +159,14 @@ PyObject * getPyAttribute(fapi2::AttributeData & i_data)
 %pythoncode %{
 def fapi2GetAttr(i_target, i_id):
     (rc, attr_id) = fapi2AttributeStringToId(i_id)
+    if (rc):
+        return None
```

Comment:
Can you make this function always return a tuple of (rc, None) in case of an error? That way the rc wouldn't get lost and pyecmd could convert it into an exception.

Reply:
done